Cargo.toml

```
@@ -21,10 +21,11 @@ futures-util = "0.3.5"
 http = "0.2.1"
 hyper = "0.13.5"
 hyper-tls = { version = "0.4.1", optional = true }
-ruma-api = "0.16.1"
-ruma-client-api = "0.9.0"
-ruma-events = "0.21.2"
-ruma-identifiers = "0.16.1"
+ruma-api = "0.17.0-alpha.1"
```

The alpha releases need to be exact version requirements, otherwise things will break when `alpha.2` is released.

```suggestion
ruma-api = "=0.17.0-alpha.1"
```